src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
`grunt lint` - but it's possible I'm having a versioning issue with the node_modules that enable the linter or otherwise (it wouldn't be the first time).